### ISSUE TITLE
Install on Ionic App

### ISSUE THREAD

**Issue by brunomonteiro3** [siddii/angular-timer#217]
Hello,

I have tried several ways to install angular-timer on a Ionic app, but with no success. Any idea on how to do it or if it's even possible?

Many thanks.

**Comment by siddii** [comment]
That's on my list to test. Can you create a plunkr or something so I can take a look at it?

**Comment by brunomonteiro3** [comment]
No, I don't have any preview published. Sorry about that.

But for testing, I have just started a blank Ionic project, installed all the dependencies required by angular-timer, but when I try to start the timer, it doesn't even appear on the page.

**Comment by siddii** [comment]
any console error?

**Comment by brunomonteiro3** [comment]
I got this error, but I'm not sure if this is related. 

http://i.imgur.com/qwPrBK8.png

**Comment by siddii** [comment]
Do you have this dependencies installed - https://github.com/siddii/angular-timer#requirements ?

**Comment by brunomonteiro3** [comment]
Yes! I have installed them via Bower and as you can see my code is requesting these files correctly: http://i.imgur.com/nVDaPIL.png

**Comment by siddii** [comment]
All I can say is that its something related to how angular-timer pulls in momentjs & humanize-duration dependencies. I have been thinking of breaking that dependency (making it optional) for a while. 

I'll take a look at it when I get a chance.

**Comment by brunomonteiro3** [comment]
All right. Thanks for your help!

**Comment by siddii** [comment]
The current code on master should be able to support Ionic. Please checkout #219

**Comment by DaDanny** [comment]
I was able to get it working just fine by running bower install angular-timer --save

*(Make sure you have the save flag so it will update your bower.json file.)*

Also, the order you are referencing the files could cause an error. I have this at the end of my bower script reference in index.html

```javascript
    <script src="bower_components/momentjs/moment.js"></script>
    <script src="bower_components/humanize-duration/humanize-duration.js"></script>
    <script src="bower_components/angular-timer/dist/angular-timer.js"></script>
```
so these are loaded after angular and ionic. 

Also, are you adding 'timer' to your app dependencies?

**Comment by SvOzMaS** [comment]
DaDanny solution works for me. :)

**Comment by zvineyard** [comment]
@DaDanny's solution also worked for me, but bower added my files to these paths:

```
<script src="lib/moment/moment.js"></script>
<script src="lib/humanize-duration/humanize-duration.js"></script>
<script src="lib/angular-timer/dist/angular-timer.js"></script>
```

**Comment by DaDanny** [comment]
@zvineyard That should be fine. Ionic uses the lib directory out of the box, so if you built your app using ionic start, that is the expected behavior. I just preferred to keep it in my bower_components directory.

Glad I could help though. Let me know if you have any more trouble.
